pkg/machine/qemu/command/command.go

```diff
@@ -52,10 +52,19 @@ func (q *QemuCmd) SetQmpMonitor(monitor Monitor) {
 }
 
 // SetNetwork adds a network device to the machine
-func (q *QemuCmd) SetNetwork() {
+func (q *QemuCmd) SetNetwork(vlanSocket *define.VMFile) error {
 	// Right now the mac address is hardcoded so that the host networking gives it a specific IP address.  This is
 	// why we can only run one vm at a time right now
-	*q = append(*q, "-netdev", "socket,id=vlan,fd=3", "-device", "virtio-net-pci,netdev=vlan,mac=5a:94:ef:e4:0c:ee")
+	if vlanSocket == nil {
+		return errors.New("vlanSocket is undefined")
+	}
+	*q = append(*q, []string{"-netdev", socketVlanNetdev(vlanSocket.GetPath())}...)
+	*q = append(*q, []string{"-device", "virtio-net-pci,netdev=vlan,mac=5a:94:ef:e4:0c:ee"}...)
```

Comment:
Need to change this to remove old copy paste of the arrays, not needed any more:
```suggestion
	*q = append(*q, "-netdev", socketVlanNetdev(vlanSocket.GetPath()))
	*q = append(*q, "-device", "virtio-net-pci,netdev=vlan,mac=5a:94:ef:e4:0c:ee")
```